ui/ducks/send/send.js

```diff
@@ -342,7 +347,7 @@ export const computeEstimatedGasLimit = createAsyncThunk(
     if (send.stage !== SEND_STAGES.EDIT) {
       const gasLimit = await estimateGasLimitForSend({
         gasPrice: send.gas.gasPrice,
-        blockGasLimit: metamask.blockGasLimit,
```

Comment:
We should investigate the difference between `blockGasLimit` and `currentBlockGasLimit` and why we were using something that was seemingly not defined. We don't necessarily have to do this before this PR is merged, but I would like to create an issue, and block release on answering the question, to make sure there are no other related bugs, or no important differences between what we expected `blockGasLimit` to be and what `currentBlockGasLimit` is

Reply:
It most likely happened as a result of the send flow refactor where we were taking currentBlockGasLimit from state and passing it to utility methods as a 'blockGasLimit' prop and then I improperly selected it from state.

Reply:
currentBlockGasLimit is the legitimate value. A prop named blockGasLimit is used in the tx-gas-utils and transaction controller, but its set in state in the `account-tracker` as `currentBlockGasLimit`

```
  async _updateForBlock(blockNumber) {
    this._currentBlockNumber = blockNumber;

    // block gasLimit polling shouldn't be in account-tracker shouldn't be here...
    const currentBlock = await this._query.getBlockByNumber(blockNumber, false);
    if (!currentBlock) {
      return;
    }
    const currentBlockGasLimit = currentBlock.gasLimit;
    this.store.updateState({ currentBlockGasLimit });

    try {
      await this._updateAccounts();
    } catch (err) {
      log.error(err);
    }
  }
```